Adds support for two-way authentication using .cer and .pem files.

ppcuban opened the pull request.

danielkza commented on README.rst:
```diff
@@ -105,7 +105,12 @@ Common parameters:
+      -k KEYFILE, --keyfile KEYFILE
+                            File path of optional keyfile (.key) to be used to identify the local side of the
+                            connection.
+      -crt CERTFILE, --certfile CERTFILE
```
The short option (`-crt`) needs to be one-character only. I also think making the long names more explicit would be better, and even omitting the short name completely would be fine. What about `--ssl-client-private-key` and `--ssl-client-cert` instead?

ppcuban commented:
@danielkza , we have updated the `-crt` option to `-t` and made the long option names more descriptive. We also updated the README. (sorry about the many commits).

danielkza commented:
@ppcuban Thank you! The ammount of commits is no problem at all, it even makes the changes easier to review one-by-one.

I just have one more small request. Changing the host cert option name after a version has already been released isn't very friendly to anyone that already started using it. Can you change it (only the host one) back to how it was? Sorry if I didn't communicate that clearly enough.

ppcuban commented:
@danielkza You're absolutely right. We actually discussed that yesterday when we realized that the original `--ssl-cert` was really misleading from a functionality perspective but for some reason, we changed it inadvertently and commit.

Would you like to keep the 'help' section? I think is more descriptive of what `--ssl-cert` really does.

danielkza commented:
> Would you like to keep the 'help' section? I think is more descriptive of what --ssl-cert really does.

@ppcuban Sure, your description is much clearer than the previous one.

danielkza commented:
Thank you very much for your contribution! I'll prepare a new release as soon as I can.

wanderanimrod commented:
Thanks @danielkza

danielkza commented:
Released in version 0.2.6.
